packages/mc-dev-authentication/html-docs/login.html

```diff
@@ -0,0 +1,228 @@
+<html>
```

Comment:
As suggested by @tdeekens in another PR, we can using a plain old HTML file for these pages.

Reply:
Thanks a lot. I think this is easier at least for my lost soul :)